zhaquirks/tuya/ts0012.py

```diff
@@ -0,0 +1,110 @@
+"""Tuya 2 Button switch."""
+
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+    Time,
+)
+
+from . import (
+    TuyaManufacturerClusterOnOff,
+    TuyaManufCluster,
+    TuyaOnOff,
+    TuyaSmartRemoteOnOffCluster,
+    TuyaSwitch,
+)
+from ..const import (
+    BUTTON_1,
+    BUTTON_2,
+    COMMAND,
+    DEVICE_TYPE,
+    ENDPOINT_ID,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+)
+
+
+class TuyaDoubleSwitch(TuyaSwitch):
+    """Tuya 2-button remote device."""
+
+    signature = {
+        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=100, device_version=0, input_clusters=[0, 4, 5, 6], output_clusters=[19]))
+        # SizePrefixedSimpleDescriptor(endpoint=2, profile=260, device_type=100, device_version=0, input_clusters=[4, 5, 6], output_clusters=[])
+        MODELS_INFO: [("_TYZB01_vzrytttn", "TS0012")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    OnOff.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    Time.cluster_id,
+                    TuyaManufCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Ota.cluster_id],
+            },
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    PowerConfiguration.cluster_id,
+                    OnOff.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    Time.cluster_id,
+                    TuyaManufCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+        },
+    }
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.REMOTE_CONTROL,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    TuyaSmartRemoteOnOffCluster,
+                    Time.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    TuyaManufacturerClusterOnOff,
+                    TuyaOnOff,
+                ],
+                OUTPUT_CLUSTERS: [Ota.cluster_id],
+            },
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.REMOTE_CONTROL,
+                INPUT_CLUSTERS: [
+                    PowerConfiguration.cluster_id,
+                    TuyaSmartRemoteOnOffCluster,
```

Comment:
```suggestion
```

Reply:
It is indeed the same model I suppose. There is no neutral wire, as it is Zigbee powered, and derives a small trickle from the power wire (L).

Reply:
Logging:

2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x05\x00\x08\x78\x3D\x00\x00\x09\x09\x77\x00\x20\x00\x00\x00\x00\x00\x08\x38\x1C\x4C'
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x05\x00\x08\x78\x3D\x00\x00\x09\x09\x77\x00\x20\x00\x00\x00\x00\x00\x08\x38\x1C'))
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=5, SecurityUse=<Bool.false: 0>, TimeStamp=4028424, TSN=0, Data=b'\x09\x77\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0x3808, MsgResultRadius=28)
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=119 command_id=0>
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:06 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=5, SecurityUse=<Bool.false: 0>, TimeStamp=4028424, TSN=0, Data=b'\x09\x77\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0x3808, MsgResultRadius=28) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:06 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone, old_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone=on; friendly_name=LidlSensorWoonkamer 19ca79fe ias_zone, device_class=motion @ 2021-01-08T10:25:58.239660+01:00>, new_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone=off; friendly_name=LidlSensorWoonkamer 19ca79fe ias_zone, device_class=motion @ 2021-01-08T10:27:06.214444+01:00>>
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1C\x44\x81\x00\x00\x06\x00\xEB\x25\x01\x01\x00\x07\x00\xD9\x78\x3D\x00\x00\x08\x01\x6A\x42\x00\x08\x07\x00\x00\x12\xF8\x0A\x44'
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x06\x00\xEB\x25\x01\x01\x00\x07\x00\xD9\x78\x3D\x00\x00\x08\x01\x6A\x42\x00\x08\x07\x00\x00\x12\xF8\x0A'))
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=6, SrcAddr=0x25EB, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=7, SecurityUse=<Bool.false: 0>, TimeStamp=4028633, TSN=0, Data=b'\x01\x6A\x42\x00\x08\x07\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=10)
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0x25eb:1:0x0006] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=106 command_id=66>
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0x25eb:1:0x0006] ZCL request 0x0042: [0, 1800, 0]
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy.zcl] [0x25eb:1:0x0006] No handler for cluster command 66
2021-01-08 10:27:06 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event zha_event[L]: device_ieee=68:0a:e2:ff:fe:38:e7:f0, unique_id=68:0a:e2:ff:fe:38:e7:f0:1:0x0006, device_id=f0bdb3de1139f0f079936b3b5720d582, endpoint_id=1, cluster_id=6, command=on_with_timed_off, args=[0, 1800, 0]>
2021-01-08 10:27:06 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x25EB:1:0x0006]: received 'on_with_timed_off' command with [0, 1800, 0] args on cluster_id '6' tsn '106'
2021-01-08 10:27:06 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=6, SrcAddr=0x25EB, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=7, SecurityUse=<Bool.false: 0>, TimeStamp=4028633, TSN=0, Data=b'\x01\x6A\x42\x00\x08\x07\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=10) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:06 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=binary_sensor.ikea_of_sweden_tradfri_motion_sensor_f0e738fe_on_off, old_state=<state binary_sensor.ikea_of_sweden_tradfri_motion_sensor_f0e738fe_on_off=off; friendly_name=IKEA of Sweden TRADFRI motion sensor f0e738fe on_off, device_class=motion @ 2021-01-08T10:26:45.335282+01:00>, new_state=<state binary_sensor.ikea_of_sweden_tradfri_motion_sensor_f0e738fe_on_off=on; friendly_name=IKEA of Sweden TRADFRI motion sensor f0e738fe on_off, device_class=motion @ 2021-01-08T10:27:06.281216+01:00>>
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0F\x00\x9F\xB5\x3D\x00\x00\x09\x09\x78\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B\xCE'
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0F\x00\x9F\xB5\x3D\x00\x00\x09\x09\x78\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4044191, TSN=0, Data=b'\x09\x78\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=120 command_id=0>
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:11 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:11 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4044191, TSN=0, Data=b'\x09\x78\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0A\x00\x24\xF3\x3D\x00\x00\x09\x09\x79\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B\x37'
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0A\x00\x24\xF3\x3D\x00\x00\x09\x09\x79\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=10, SecurityUse=<Bool.false: 0>, TimeStamp=4059940, TSN=0, Data=b'\x09\x79\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=121 command_id=0>
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:16 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:16 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=10, SecurityUse=<Bool.false: 0>, TimeStamp=4059940, TSN=0, Data=b'\x09\x79\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1B\x44\x81\x00\x00\x00\x00\x19\x37\x0B\x01\x00\x0D\x00\x67\x02\x3E\x00\x00\x07\x18\x07\x0A\x01\x00\x20\x42\x12\xF8\x1B\x2C'
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x00\x19\x37\x0B\x01\x00\x0D\x00\x67\x02\x3E\x00\x00\x07\x18\x07\x0A\x01\x00\x20\x42\x12\xF8\x1B'))
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=0, SrcAddr=0x3719, SrcEndpoint=11, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4063847, TSN=0, Data=b'\x18\x07\x0A\x01\x00\x20\x42', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy.zcl] [0x3719:11:0x0000] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=False disable_default_response=True> manufacturer=None tsn=7 command_id=Command.Report_Attributes>
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy.zcl] [0x3719:11:0x0000] ZCL request 0x000a: [[Attribute(attrid=1, value=<TypeValue type=uint8_t, value=66>)]]
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy.zcl] [0x3719:11:0x0000] Attribute report received: app_version=66
2021-01-08 10:27:17 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=0, SrcAddr=0x3719, SrcEndpoint=11, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4063847, TSN=0, Data=b'\x18\x07\x0A\x01\x00\x20\x42', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:21 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x15\x00\x22\x38\x3E\x00\x00\x09\x09\x7A\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B\xE5'
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x15\x00\x22\x38\x3E\x00\x00\x09\x09\x7A\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=21, SecurityUse=<Bool.false: 0>, TimeStamp=4077602, TSN=0, Data=b'\x09\x7A\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=122 command_id=0>
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:21 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:21 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=21, SecurityUse=<Bool.false: 0>, TimeStamp=4077602, TSN=0, Data=b'\x09\x7A\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x0D\x45\xC1\x06\xA4\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80\xE8'
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC1, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80'))
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:22 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device announce: ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:22 INFO (MainThread) [zigpy.application] Device 0xa406 (ec:1b:bd:ff:fe:79:ca:19) joined the network
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy.application] Skip initialization for existing device ec:1b:bd:ff:fe:79:ca:19
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy_znp.zigbee.application] Pretending we received a ZDO message: b'\xFF\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80'
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy.zdo] [0xa406:zdo] ZDO request ZDOCmd.Device_annce: [0xA406, ec:1b:bd:ff:fe:79:ca:19, 128]
2021-01-08 10:27:22 DEBUG (MainThread) [zigpy_znp.api] ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>) matches CallbackResponseListener(matching_commands=(ZDO.EndDeviceAnnceInd.Callback(Src=None, NWK=None, IEEE=None, Capabilities=None),), callback=<bound method ControllerApplication.on_zdo_device_announce of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:23 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xEF\x49\x3E\x00\x00\x09\x09\x70\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1A\x38'
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xEF\x49\x3E\x00\x00\x09\x09\x70\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4082159, TSN=0, Data=b'\x09\x70\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=112 command_id=0>
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:23 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:23 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4082159, TSN=0, Data=b'\x09\x70\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:23 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone, old_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone=on; friendly_name=LidlSensor Kantoor ias_zone, device_class=motion @ 2021-01-08T10:25:37.411791+01:00>, new_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone=off; friendly_name=LidlSensor Kantoor ias_zone, device_class=motion @ 2021-01-08T10:27:23.401211+01:00>>
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:26 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1B\x44\x81\x00\x00\x01\x00\x06\xA4\x01\x01\x00\x15\x00\x3E\x73\x3E\x00\x00\x07\x08\x7B\x0A\x21\x00\x20\xC0\x12\xF8\x1B\x55'
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x01\x00\x06\xA4\x01\x01\x00\x15\x00\x3E\x73\x3E\x00\x00\x07\x08\x7B\x0A\x21\x00\x20\xC0\x12\xF8\x1B'))
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=21, SecurityUse=<Bool.false: 0>, TimeStamp=4092734, TSN=0, Data=b'\x08\x7B\x0A\x21\x00\x20\xC0', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=123 command_id=Command.Report_Attributes>
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] ZCL request 0x000a: [[Attribute(attrid=33, value=<TypeValue type=uint8_t, value=192>)]]
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] Attribute report received: battery_percentage_remaining=192
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=21, SecurityUse=<Bool.false: 0>, TimeStamp=4092734, TSN=0, Data=b'\x08\x7B\x0A\x21\x00\x20\xC0', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=123),), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436]>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Sending request: AF.DataRequestExt.Req(DstAddrModeAddress=AddrModeAddress(mode=<AddrMode.NWK: 2>, address=0xA406), DstEndpoint=1, DstPanId=0x0000, SrcEndpoint=1, ClusterId=1, TSN=123, Options=<TransmitOptions.SUPPRESS_ROUTE_DISC_NETWORK: 32>, Radius=30, Data=b'\x18\x7B\x0B\x0A\x00')
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436, <TaskWakeupMethWrapper object at 0x7f9cdf3c40>()]>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Sending data: b'\xFE\x19\x24\x02\x02\x06\xA4\x00\x00\x00\x00\x00\x00\x01\x00\x00\x01\x01\x00\x7B\x20\x1E\x05\x00\x18\x7B\x0B\x0A\x00\xBC'
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x01\x64\x02\x00\x67'
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP), data=b'\x00'))
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>) matches OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x00, subsystem=Subsystem.RPCError, type=CommandType.SRSP)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP)
2021-01-08 10:27:26 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 1 one-shot listeners remaining
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x03\x44\x80\x00\x01\x7B\xBD'
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x01\x7B'))
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=123)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=123) matches OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=123),), future=<Future finished result=AF.DataConfir...nt=1, TSN=123)>)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=123),), future=<Future finished result=AF.DataConfir...nt=1, TSN=123)>)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=123),), future=<Future finished result=AF.DataConfir...nt=1, TSN=123)>)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:27 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining
2021-01-08 10:27:27 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=sun.sun, old_state=<state sun.sun=above_horizon; next_dawn=2021-01-09T07:06:56+00:00, next_dusk=2021-01-08T16:26:31+00:00, next_midnight=2021-01-08T23:47:35+00:00, next_noon=2021-01-08T11:46:56+00:00, next_rising=2021-01-09T07:47:11+00:00, next_setting=2021-01-08T15:46:09+00:00, elevation=9.44, azimuth=147.0, rising=True, friendly_name=Sun @ 2021-01-08T10:05:27.366740+01:00>, new_state=<state sun.sun=above_horizon; next_dawn=2021-01-09T07:06:56+00:00, next_dusk=2021-01-08T16:26:31+00:00, next_midnight=2021-01-08T23:47:35+00:00, next_noon=2021-01-08T11:46:56+00:00, next_rising=2021-01-09T07:47:11+00:00, next_setting=2021-01-08T15:46:09+00:00, elevation=9.6, azimuth=147.44, rising=True, friendly_name=Sun @ 2021-01-08T10:05:27.366740+01:00>>
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:28 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0D\x00\x86\x87\x3E\x00\x00\x09\x09\x71\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1A\x9C'
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0D\x00\x86\x87\x3E\x00\x00\x09\x09\x71\x00\x20\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4097926, TSN=0, Data=b'\x09\x71\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=113 command_id=0>
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports: 32>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:28 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.0
2021-01-08 10:27:28 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4097926, TSN=0, Data=b'\x09\x71\x00\x20\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:30 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xE2\xA4\x3E\x00\x00\x09\x09\x72\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A\xDB'
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xE2\xA4\x3E\x00\x00\x09\x09\x72\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4105442, TSN=0, Data=b'\x09\x72\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=114 command_id=0>
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:30 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:30 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4105442, TSN=0, Data=b'\x09\x72\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:30 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone, old_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone=off; friendly_name=LidlSensor Kantoor ias_zone, device_class=motion @ 2021-01-08T10:27:23.401211+01:00>, new_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_ias_zone=on; friendly_name=LidlSensor Kantoor ias_zone, device_class=motion @ 2021-01-08T10:27:30.856958+01:00>>
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:35 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0F\x00\x3C\xE1\x3E\x00\x00\x09\x09\x7C\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B\x3F'
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0F\x00\x3C\xE1\x3E\x00\x00\x09\x09\x7C\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4120892, TSN=0, Data=b'\x09\x7C\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=124 command_id=0>
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:35 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4120892, TSN=0, Data=b'\x09\x7C\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:35 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event state_changed[L]: entity_id=binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone, old_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone=off; friendly_name=LidlSensorWoonkamer 19ca79fe ias_zone, device_class=motion @ 2021-01-08T10:27:06.214444+01:00>, new_state=<state binary_sensor.tz1800_fcdjzz3s_ty0202_19ca79fe_ias_zone=on; friendly_name=LidlSensorWoonkamer 19ca79fe ias_zone, device_class=motion @ 2021-01-08T10:27:35.805840+01:00>>
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:35 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\x66\xE2\x3E\x00\x00\x09\x09\x73\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A\x18'
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\x66\xE2\x3E\x00\x00\x09\x09\x73\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4121190, TSN=0, Data=b'\x09\x73\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=115 command_id=0>
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:35 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:35 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4121190, TSN=0, Data=b'\x09\x73\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:40 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0D\x00\xD6\x1E\x3F\x00\x00\x09\x09\x7D\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B\x28'
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0D\x00\xD6\x1E\x3F\x00\x00\x09\x09\x7D\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4136662, TSN=0, Data=b'\x09\x7D\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=125 command_id=0>
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:40 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4136662, TSN=0, Data=b'\x09\x7D\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:40 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xF8\x1F\x3F\x00\x00\x09\x09\x74\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A\x7D'
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\xF8\x1F\x3F\x00\x00\x09\x09\x74\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4136952, TSN=0, Data=b'\x09\x74\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=116 command_id=0>
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:40 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:40 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4136952, TSN=0, Data=b'\x09\x74\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:45 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x07\x00\x5C\x5C\x3F\x00\x00\x09\x09\x7E\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B\xE9'
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x07\x00\x5C\x5C\x3F\x00\x00\x09\x09\x7E\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=7, SecurityUse=<Bool.false: 0>, TimeStamp=4152412, TSN=0, Data=b'\x09\x7E\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=126 command_id=0>
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:45 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:45 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=7, SecurityUse=<Bool.false: 0>, TimeStamp=4152412, TSN=0, Data=b'\x09\x7E\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x19\x37\x02\x08\x38\x12\xF8\x70'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x19\x37\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812])
2021-01-08 10:27:46 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812])
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x19\x37\x02\x08\x38\x12\xF8\x70'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x19\x37\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812])
2021-01-08 10:27:46 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812])
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x3719, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:46 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x0D\x45\xC1\x4E\x9C\x4E\x9C\xCD\xA2\x70\xFE\xFF\xBD\x1B\xEC\x80\x5D'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC1, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x4E\x9C\xCD\xA2\x70\xFE\xFF\xBD\x1B\xEC\x80'))
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.EndDeviceAnnceInd.Callback(Src=0x9C4E, NWK=0x9C4E, IEEE=ec:1b:bd:ff:fe:70:a2:cd, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:46 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device announce: ZDO.EndDeviceAnnceInd.Callback(Src=0x9C4E, NWK=0x9C4E, IEEE=ec:1b:bd:ff:fe:70:a2:cd, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:46 INFO (MainThread) [zigpy.application] Device 0x9c4e (ec:1b:bd:ff:fe:70:a2:cd) joined the network
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy.application] Skip initialization for existing device ec:1b:bd:ff:fe:70:a2:cd
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.zigbee.application] Pretending we received a ZDO message: b'\xFF\x4E\x9C\xCD\xA2\x70\xFE\xFF\xBD\x1B\xEC\x80'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy.zdo] [0x9c4e:zdo] ZDO request ZDOCmd.Device_annce: [0x9C4E, ec:1b:bd:ff:fe:70:a2:cd, 128]
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] ZDO.EndDeviceAnnceInd.Callback(Src=0x9C4E, NWK=0x9C4E, IEEE=ec:1b:bd:ff:fe:70:a2:cd, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>) matches CallbackResponseListener(matching_commands=(ZDO.EndDeviceAnnceInd.Callback(Src=None, NWK=None, IEEE=None, Capabilities=None),), callback=<bound method ControllerApplication.on_zdo_device_announce of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\x71\x64\x3F\x00\x00\x09\x09\x75\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A\x8E'
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x4E\x9C\x01\x01\x00\x0F\x00\x71\x64\x3F\x00\x00\x09\x09\x75\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1A'))
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4154481, TSN=0, Data=b'\x09\x75\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=117 command_id=0>
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:46 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0x9C4E:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:46 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4154481, TSN=0, Data=b'\x09\x75\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x05\x45\xC4\x08\x38\x01\x12\xF8\x5F'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x08\x38\x01\x12\xF8'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812])
2021-01-08 10:27:51 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812])
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x05\x45\xC4\x08\x38\x01\x12\xF8\x5F'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x08\x38\x01\x12\xF8'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812])
2021-01-08 10:27:51 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812])
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x3808, Relays=[0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x09\x45\xC4\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8\xAD'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x4E\x9C\x03\x19\x37\x08\x38\x12\xF8'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:51 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812])
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0x9C4E, Relays=[0x3719, 0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1B\x44\x81\x00\x00\x01\x00\x4E\x9C\x01\x01\x00\x0F\x00\xA1\x9F\x3F\x00\x00\x07\x08\x76\x0A\x21\x00\x20\xC8\x12\xF8\x1A\x49'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x01\x00\x4E\x9C\x01\x01\x00\x0F\x00\xA1\x9F\x3F\x00\x00\x07\x08\x76\x0A\x21\x00\x20\xC8\x12\xF8\x1A'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4169633, TSN=0, Data=b'\x08\x76\x0A\x21\x00\x20\xC8', MacSrcAddr=0xF812, MsgResultRadius=26)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=118 command_id=Command.Report_Attributes>
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0001] ZCL request 0x000a: [[Attribute(attrid=33, value=<TypeValue type=uint8_t, value=200>)]]
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0x9c4e:1:0x0001] Attribute report received: battery_percentage_remaining=200
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0x9C4E, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4169633, TSN=0, Data=b'\x08\x76\x0A\x21\x00\x20\xC8', MacSrcAddr=0xF812, MsgResultRadius=26) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=118),), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436]>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Sending request: AF.DataRequestExt.Req(DstAddrModeAddress=AddrModeAddress(mode=<AddrMode.NWK: 2>, address=0x9C4E), DstEndpoint=1, DstPanId=0x0000, SrcEndpoint=1, ClusterId=1, TSN=118, Options=<TransmitOptions.SUPPRESS_ROUTE_DISC_NETWORK: 32>, Radius=30, Data=b'\x18\x76\x0B\x0A\x00')
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436, <TaskWakeupMethWrapper object at 0x7f9b2bb190>()]>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Sending data: b'\xFE\x19\x24\x02\x02\x4E\x9C\x00\x00\x00\x00\x00\x00\x01\x00\x00\x01\x01\x00\x76\x20\x1E\x05\x00\x18\x76\x0B\x0A\x00\xCC'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x01\x64\x02\x00\x67'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP), data=b'\x00'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>) matches OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x00, subsystem=Subsystem.RPCError, type=CommandType.SRSP)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 1 one-shot listeners remaining
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x03\x44\x80\x00\x01\x76\xB0'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x01\x76'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=118)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=118) matches OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=118),), future=<Future finished result=AF.DataConfir...nt=1, TSN=118)>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=118),), future=<Future finished result=AF.DataConfir...nt=1, TSN=118)>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=118),), future=<Future finished result=AF.DataConfir...nt=1, TSN=118)>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x0D\x45\xC1\x06\xA4\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80\xE8'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC1, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:51 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device announce: ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>)
2021-01-08 10:27:51 INFO (MainThread) [zigpy.application] Device 0xa406 (ec:1b:bd:ff:fe:79:ca:19) joined the network
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.application] Skip initialization for existing device ec:1b:bd:ff:fe:79:ca:19
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.zigbee.application] Pretending we received a ZDO message: b'\xFF\x06\xA4\x19\xCA\x79\xFE\xFF\xBD\x1B\xEC\x80'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zdo] [0xa406:zdo] ZDO request ZDOCmd.Device_annce: [0xA406, ec:1b:bd:ff:fe:79:ca:19, 128]
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] ZDO.EndDeviceAnnceInd.Callback(Src=0xA406, NWK=0xA406, IEEE=ec:1b:bd:ff:fe:79:ca:19, Capabilities=<MACCapabilities.AllocateShortAddrDuringAssocNeeded: 128>) matches CallbackResponseListener(matching_commands=(ZDO.EndDeviceAnnceInd.Callback(Src=None, NWK=None, IEEE=None, Capabilities=None),), callback=<bound method ControllerApplication.on_zdo_device_announce of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:51 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1D\x44\x81\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0D\x00\xD1\xA0\x3F\x00\x00\x09\x09\x7F\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B\x93'
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x00\x05\x06\xA4\x01\x01\x00\x0D\x00\xD1\xA0\x3F\x00\x00\x09\x09\x7F\x00\x21\x00\x00\x00\x00\x00\x12\xF8\x1B'))
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4169937, TSN=0, Data=b'\x09\x7F\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=CLUSTER_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=127 command_id=0>
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] ZCL request 0x0000: [<ZoneStatus.Restore_reports|Alarm_1: 33>, <bitmap8.0: 0>, 0, 0]
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0500] No handler for cluster command 0
2021-01-08 10:27:51 DEBUG (MainThread) [homeassistant.components.zha.core.channels.base] [0xA406:1:0x0500]: Updated alarm state: ZoneStatus.Alarm_1
2021-01-08 10:27:51 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1280, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=13, SecurityUse=<Bool.false: 0>, TimeStamp=4169937, TSN=0, Data=b'\x09\x7F\x00\x21\x00\x00\x00\x00\x00', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x07\x45\xC4\x06\xA4\x02\x08\x38\x12\xF8\xFC'
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0xC4, subsystem=Subsystem.ZDO, type=CommandType.AREQ), data=b'\x06\xA4\x02\x08\x38\x12\xF8'))
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Received command: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:56 INFO (MainThread) [zigpy_znp.zigbee.application] ZDO device relays: ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812])
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] ZDO.SrcRtgInd.Callback(DstAddr=0xA406, Relays=[0x3808, 0xF812]) matches CallbackResponseListener(matching_commands=(ZDO.SrcRtgInd.Callback(DstAddr=None, Relays=None),), callback=<bound method ControllerApplication.on_zdo_relays_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x1B\x44\x81\x00\x00\x01\x00\x06\xA4\x01\x01\x00\x0F\x00\xEE\xDB\x3F\x00\x00\x07\x08\x00\x0A\x21\x00\x20\xC0\x12\xF8\x1B\x4D'
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x81, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x00\x01\x00\x06\xA4\x01\x01\x00\x0F\x00\xEE\xDB\x3F\x00\x00\x07\x08\x00\x0A\x21\x00\x20\xC0\x12\xF8\x1B'))
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4185070, TSN=0, Data=b'\x08\x00\x0A\x21\x00\x20\xC0', MacSrcAddr=0xF812, MsgResultRadius=27)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] ZCL deserialize: <ZCLHeader frame_control=<FrameControl frame_type=GLOBAL_COMMAND manufacturer_specific=False is_reply=False disable_default_response=False> manufacturer=None tsn=0 command_id=Command.Report_Attributes>
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] ZCL request 0x000a: [[Attribute(attrid=33, value=<TypeValue type=uint8_t, value=192>)]]
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy.zcl] [0xa406:1:0x0001] Attribute report received: battery_percentage_remaining=192
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] AF.IncomingMsg.Callback(GroupId=0x0000, ClusterId=1, SrcAddr=0xA406, SrcEndpoint=1, DstEndpoint=1, WasBroadcast=<Bool.false: 0>, LQI=15, SecurityUse=<Bool.false: 0>, TimeStamp=4185070, TSN=0, Data=b'\x08\x00\x0A\x21\x00\x20\xC0', MacSrcAddr=0xF812, MsgResultRadius=27) matches CallbackResponseListener(matching_commands=(AF.IncomingMsg.Callback(GroupId=None, ClusterId=None, SrcAddr=None, SrcEndpoint=None, DstEndpoint=None, WasBroadcast=None, LQI=None, SecurityUse=None, TimeStamp=None, TSN=None, Data=None, MacSrcAddr=None, MsgResultRadius=None),), callback=<bound method ControllerApplication.on_af_message of <zigpy_znp.zigbee.application.ControllerApplication object at 0x7fa158b640>>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=0),), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436]>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Sending request: AF.DataRequestExt.Req(DstAddrModeAddress=AddrModeAddress(mode=<AddrMode.NWK: 2>, address=0xA406), DstEndpoint=1, DstPanId=0x0000, SrcEndpoint=1, ClusterId=1, TSN=0, Options=<TransmitOptions.SUPPRESS_ROUTE_DISC_NETWORK: 32>, Radius=30, Data=b'\x18\x00\x0B\x0A\x00')
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Creating one-shot listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future pending cb=[ZNP.wait_for_responses.<locals>.<lambda>() at /usr/local/lib/python3.8/site-packages/zigpy_znp/api.py:436, <TaskWakeupMethWrapper object at 0x7f9bcb55b0>()]>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Sending data: b'\xFE\x19\x24\x02\x02\x06\xA4\x00\x00\x00\x00\x00\x00\x01\x00\x00\x01\x01\x00\x00\x20\x1E\x05\x00\x18\x00\x0B\x0A\x00\xBC'
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x01\x64\x02\x00\x67'
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP), data=b'\x00'))
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] AF.DataRequestExt.Rsp(Status=<Status.SUCCESS: 0>) matches OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataRequestExt.Rsp(Status=None), RPCError.CommandNotRecognized.Rsp(ErrorCode=None, RequestHeader=CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SREQ))), future=<Future finished result=AF.DataReques...s.SUCCESS: 0>)>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x00, subsystem=Subsystem.RPCError, type=CommandType.SRSP)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x02, subsystem=Subsystem.AF, type=CommandType.SRSP)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 1 one-shot listeners remaining
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Received data: b'\xFE\x03\x44\x80\x00\x01\x00\xC6'
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.uart] Parsed frame: TransportFrame(payload=GeneralFrame(header=CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ), data=b'\x00\x01\x00'))
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Received command: AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=0)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] AF.DataConfirm.Callback(Status=<Status.SUCCESS: 0>, Endpoint=1, TSN=0) matches OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=0),), future=<Future finished result=AF.DataConfir...oint=1, TSN=0)>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=0),), future=<Future finished result=AF.DataConfir...oint=1, TSN=0)>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Removing listener OneShotResponseListener(matching_commands=(AF.DataConfirm.Callback(Status=None, Endpoint=None, TSN=0),), future=<Future finished result=AF.DataConfir...oint=1, TSN=0)>)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] Cleaning up empty listener list for header CommandHeader(id=0x80, subsystem=Subsystem.AF, type=CommandType.AREQ)
2021-01-08 10:27:56 DEBUG (MainThread) [zigpy_znp.api] There are 5 callbacks and 0 one-shot listeners remaining